package.json

```diff
@@ -11,7 +11,8 @@
     "lab": "5.x.x"
   },
   "scripts": {
-    "test": "make test-cov",
+    "test": "node node_modules/lab/bin/lab -a code -t 100 -L",
```

Comment:
Yeah right, bad memory from trying to make lab work on windows the other day, I had to do that, my bad :)